msteams-platform/graph-permissions/resource-specific-consent.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Resource specific consent in Teams
+description: Describes resource specific consent in Teams and how to make advantage of it.
+localization_priority:  Normal
+author: laujan
+ms.author: lajanuar
+ms.topic: Overview
+keywords: teams authorization OAuth SSO AAD rsc Graph
+---
+# Resource specific consent (RSC) — Microsoft Teams Graph API
+
+Microsoft Teams and Graph API integration enables you to use API endpoints to manage team life cycles within their organizations. When originally introduced, the permissions model was non-granular and required the approval of a global admin. The resource specific consent (RSC) permissions model enables *team owners* to grant consent for an application to access and/or modify a team's data. The granular, Teams-specific, RSC permissions define what an application can do within a specific team:
+
+![team lifecycle](../assets/images/team-lifecycle.svg)
+</br></br>
+
+|Application Permission| Action |
+| ----- | ----- |
+|TeamSettings.Read.Group | Get the settings for this team.|
+|TeamSettings.Edit.Group|Update the settings for this team.|
+|ChannelSettings.Read.Group|Get the channel names, channel descriptions, and channel settings for this team​.|
+|ChannelSettings.Edit.Group|Update the channel names, channel descriptions, and channel settings for this team.​|
+|Channel.Create.Group|Create channels in this team.​|
+|Channel.Delete.Group|Delete channels in this team.​|
+|ChannelMessage.Read.Group |Get this team's channel messages.​|
+|TeamsApp.Read.Group|Get a list of this team's installed apps.|
+|TeamsTab.Read.Group|Get a list of this team's tabs.|
+|TeamsTab.Create.Group|Create tabs in this team.​|
+|TeamsTab.Edit.Group|Update this team's tabs.​|
+|TeamsTab.Delete.Group|Delete this team's tabs.​|
+|Member.Read.Group|Get this team's members.​|
+|Owner.Read.Group|Get this team's owners.​|
+|Member.ReadWrite.Group|Add and remove members from this team.​|
+|Owner.ReadWrite.Group|Add and remove owners from this team.​|
+|File.Read.Group|Open a file,get all text, and close the file in this team.|
+|File.Create.Group|Create or open a file in this team. Note: if the file already exists, its contents will be overwritten.|
+|File.Edit.Group|Update the name of a specified file in this team.|
+|File.Delete.Group|Delete a specified file in this team.|
+
+>[!NOTE]
+>Resource specific permissions are only available to Teams apps installed on the Teams client and are currently not part of Azure AD.
+
+### **Enabling RSC in your application** 
+
+## 1. Ensure that the Tenant admin has configured [group owner consent](/azure/active-directory/manage-apps/configure-user-consent#configure-group-owner-consent-to-apps-accessing-group-data) settings in Azure Active Directory (Azure AD)
+
+- **Disable or enable group owner consent from the Azure portal**  
```

Comment:
Its not clear from reading this that the power shell is an alternative to the UI above